node/src/main/java/io/airlift/node/NodeInfo.java

```diff
@@ -249,6 +273,15 @@ public long getStartTime()
         return startTime;
     }
 
+    /**
+     * Annotations describing this server and/or the environment in which it is running.
+     */
+    @Managed
```

Comment:
Is it safe to tag a map as managed?

Reply:
I don't think it'll export properly, you want to use this as labels I think, not as a time series value

Reply:
Thanks, removed.